src/content/guide/how-to-build-a-product/security.md

```diff
@@ -0,0 +1,176 @@
+---
+title: Security
+columns: two
+template: guide.hbs
+order: 5
+---
+
+# Security
+
+When it comes to IoT devices, there is no shortage of security concerns. This
+part of the guide will be a discussion of security-related topics that arise
+with a product at scale.
+
+## Treating Devices as Products
+
+When you created a product, you created a new type of device in the Particle
```

Comment:
You can link to the previous docs section `[When you created a product](/guide/how-to-build-a-product/dashboard/#defining-a-product)`